src/client.ts

```diff
@@ -183,4 +194,146 @@ export class Client {
   public getUrl(path: string = ''): string {
     return this.transport.getUrl(path);
   }
+
+  /**
+   * Downloads a file from the store
+   *
+   * The file is placed in the download directory. If the file does not exist,
+   * an error is thrown.
+   *
+   * @param artifact A file object to download
+   * @returns Absolute path to the local copy of the file
+   * @async
```

Comment:
_nit_: The `@async` tags are redundant, as this is declared in the typescript signature.